source/documentation/deploying_apps/deploying_rails.md

```diff
@@ -64,9 +64,9 @@ You can add another instance of your app by running:
 
 ``cf scale APPNAME -i 2``
 
-###Deploying with a PostgreSQL database
+###Deploying with a database
```

Comment:
Renaming this section will break link. It doesn't look like we link to it within our own docs though.

The content is still specific about Postgres though. Should we rename it? Does it need some introduction to say that it uses Postgres as an example?